src/store/index.ts

```diff
@@ -162,10 +162,26 @@ export const indexStore = createPartialStore<IndexStoreTypes>({
 
       const allCharacterInfos = getters.GET_ALL_CHARACTER_INFOS;
 
-      // デフォルトスタイルが設定されていない場合は0をセットする
+      // デフォルトスタイルが設定されていない、または
+      // デフォルトスタイルのスタイルが存在しない場合は0をセットする
```

Comment:
この処理にしないとどこかが表示エラーになる感じでしょうか 👀

このままだとたぶんデフォルトスタイル選択ダイアログを開いたときに、初期化されたデフォルトスタイルが保存されることになると思います！
かなりレアケースですが、ユーザーにとって予想外の動きなので、FIXMEコメントを書いておくのはどうでしょう。

```suggestion
      // デフォルトスタイルが設定されていない、または
      // デフォルトスタイルのスタイルが存在しない場合は0をセットする
      // FIXME: 勝手に0番のデフォルトスタイルが保存されてしまうため、存在しないデフォルトスタイルでもUIが表示されるようにする
```